java-runtime/src/main/proto/spine/validate/validation_error.proto

```diff
@@ -65,8 +65,44 @@ message ConstraintViolation {
     // The name of the validated message type.
     string type_name = 7;
 
-    // A path to the field in an invalid value.
-    base.FieldPath field_path = 3;
+    // A field that violates the validation constraint.
+    oneof field {
+
+        // The field path.
+        //
+        // This field is populated when this violation is nested in another constraint violation.
```

Comment:
Please rephrase this sentence. There are a couple of problems here:

* "... this violation" — _this_ is not a violation, this is a validation error;
* let's describe this case: "violation is nested in another violation" — so that it is clear to readers not familiar with our internals; right now there is a gap between the fact "validation failed" and the phrase you've used.